cdc/sink/codec/avro.go

```diff
@@ -466,6 +466,8 @@ func columnToAvroNativeData(col *model.Column) (interface{}, string, error) {
 			return handleUnsignedInt64()
 		}
 		return col.Value.(int64), "long", nil
+	case mysql.TypeFloat:
+		return col.Value.(float64), "float", nil
 	default:
 		avroType, err := getAvroDataTypeFallback(col.Value)
```

Comment:
Could log the value in line avro.go:299?

```
299     log.Warn("getAvroDataTypeFallback: unknown type")
```

Reply:
Could logging the value violate customer privacy?